### ISSUE TITLE
Adding a dashboard with metrics on the Platform home page

### ISSUE THREAD

**Issue by buniello** [opentargets/issues#2653]
- As a product owner, I would like to be able to show a neat dashboard with the main Platform metrics on the Homepage - to be updated with each release.  These metrics are now only accessible from the Release Notes (though a bit hidden).

This feature will increase visibility of the Platform data to the Community. 

NOTE: the PPP will have different metrics accounting for the extra data. 

## Tasks
- [] Get number of Targets, Diseases, Evidence Strings and Associations for each release (e.g. from API)
- [] Discuss, develop and deploy a UI feature (dashboard?) that shows metrics on the Open Targets Home Page.
-
(already briefly discussed with Jarrod, Carlos and Irene)

Benchmarking: the GeneCC database has a nice[ statistics page](https://search.thegencc.org/statistics) with dashboard.

## Acceptance tests

How do we know the task is complete? 

1. Users can visualise the metrics dashboard on the homepage.

**Comment by d0choa** [comment]
This has been long-discussed and it would beneficial for the platform and its users if defined in the right form.

We had numbers such as the number of targets, diseases, evidence or associations in the past (angular app) but none of them really have any information value. For example, users have no sense of what's the meaning of 10M evidence. Also, if we decide to drastically change things (e.g. how we model epmc evidence), number of evidence might change drastically.

Instead, including information about how evidence from different data sources accumulates through time could be more meaningful. This will probably require a page/small dashboard with statistics.

There are a few steps that need to happen before we get to this:
1. Plug data-team metrics in ETL
1. Decide the relevant metrics/aggregations
1. Expose them through API
1. Build the appropriate FE.

However, we need to put this in the context of the other activities we do. I don't think we will have the capacity for such a feature before the end of the year so I will close this ticket for the purpose of streamlining the issue tracker. We will open another ticket when the time comes
